python/lsst/ctrl/mpexec/cli/script/build.py

```diff
@@ -50,6 +52,10 @@ def build(order_pipeline=None, pipeline=None, pipeline_actions=(), pipeline_dot=
         Path location for storing resulting pipeline definition in YAML format.
     show : `list` [`str`]
         Descriptions of what to dump to stdout.
+    kwargs : `dict` [`str`, `str`]
+        Ignored; click commands may accept options for more than one script
+        function and pass all the option kwargs to each of the script functions
+        which ingore these unused kwargs.
```

Comment:
Hmm, this is potentially problematic, if for example client misspells option/keyword name, you won't catch it here. Could we do something safer here (and other places like this)?

Reply:
@andy-slac I think there is not a good reason for the explicit kwargs to have default values. What if I removed them? That way if any of them are misspelled, and therefore missing, there will be an error.

Reply:
(Click generates default value for all the options, the command function gets called with a kwarg for each of the `@option`s it is decorated with.

Reply:
Having default values here and in click indeed bothers me too. If we say that all arguments are required than indeed we can drop defaults here and that should reduce the probability for spelling errors.